packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1116,7 +1116,9 @@ class TerminalHandler {
       lastReceivedCommand = command;
       await _commonTerminalInputHandler(command);
     } catch (error, st) {
-      printError('$error\n$st');
+      if (error is! ToolExit) {
```

Comment:
I added this line so that we do not spam strack traces for known error types

Reply:
That sounds like a good comment to put in the code.

Reply:
Done